### PR TITLE
Added build task

### PR DESCRIPTION
**Pull request by eddiemoore**

### CODE REVIEW

File: dist/speedo.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+'use strict';
```
Comment (sebdeckers) [inline dist/speedo.js]: Why do you want to include this build artefact?

Comment (eddiemoore) [inline dist/speedo.js]: Yeah that's what I was thinking. I probably shouldn't include it

**Comment by sebdeckers** [comment]
Do you want me to merge this or are you adding Travis/Codeship support for GH Releases?

**Comment by eddiemoore** [comment]
Can merge this for now.

**Comment by sebdeckers** [comment]
:rocket:

**Pull request merged**